### DISCUSSION TITLE
Section for invoices

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#726] [Anything Else]
This is being requested more frequently with us being increasingly used by companies and teams:

* Allow easy way to download PDF invoices in the settings
* Allow the option to send the invoices to a dedicated email address

**Update:** You can now download your invoices in your Plausible account. You can also change the billing address by opening your invoice and clicking the "Edit business address".

**Comment by metmarkosaric** [comment]
If possible this section should also have the option to change the VAT / billing info.

**Comment by metmarkosaric** [comment]
You can now download your invoices in your Plausible account. Log in to your Plausible account and in the top right menu, click on your account name and choose "Settings" to enter your account settings.

To download your invoices, find the "Invoices" section. You'll see the list of your invoices and you can download the individual invoice by clicking on the link button.

You can change the billing address by opening your invoice and clicking the "Edit business address".

If you haven't added your company details during the subscription process, you can open an invoice and click on the "Add address & VAT Number" link to do so.

**Reply by krwi** [reply]
Much appreciated! 😍

Any plans on implementing the *"send the invoices to a dedicated email address"* option?

**Reply by metmarkosaric** [reply]
thanks @krwi! we do plan to explore the Paddle API at some point to see if it's possible for us to change the email address on the Paddle account (that's where they send the emails automatically). no timeline that i can promise at this moment

**Reply by metmarkosaric** [reply]
unfortunately not possible to change an email address or set an email for invoices in Paddle API:

"Regarding an API to change email addresses, unfortunately we do not have an API enabling you to do so as email change requests are not process automatically, and require a member of our Buyer Support team to do so manually. This is because a buyer's email address can be linked to various orders across our platform (i.e., linked to another seller's orders) and so we need to confirm with the buyer that they own those other orders. However we are currently working on enabling buyers and sellers to manage email addresses more effectively, so the current logic will not be the case by the end of the year."

**Reply by krwi** [reply]
Thanks for the update!
One more reason to look forward to the end of the year 🎄
